dask/array/core.py

```diff
@@ -19,6 +19,7 @@
 from . import chunk
 from .slicing import slice_array
 from . import numpy_compat
+from ..base import DaskBase
```

Comment:
I originally wrote `Collection`, but both `dataframe` and `bag` had things representing items, so that seemed weird. Not happy with the current name either though, `Base` sounds marginally better.